views.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import messages
+from django.contrib.auth import update_session_auth_hash
+from django.shortcuts import render, redirect, get_object_or_404
+from django.utils.translation import ugettext_lazy as _
+
+from profiles.models import Address, Profile
+from profiles.forms import PasswordChangingForm, AddressForm
+
+
+# Create your views here.
+
+def my_profile(request):
```

Comment:
Please add decorator: `from django.contrib.auth.decorators import login_required` to both views as they are available only for logged in users...